api/dss.yaml

```diff
@@ -207,13 +264,6 @@ paths:
       description: Delete an Identification Service Area.  USSs should not delete
         Identification Service Areas before the end of the last managed flight plus
         the retention period.
-    parameters:
```

Comment:
Why duplicate this parameter in all three verbs?

Reply:
I got hung up on it and didn't realize what was going on with the indentation. Plus it was added in PUT which kind of confusing. I think having one block per method where we can clearly see the parameters is useful

Reply:
Sure, it's not a big deal, although I thought the goal was to not make edits to the API, whereas this leaves the API intact. Also if we can remove from PUT then we should be able to make this modification?

Reply:
done